client/src/components/Contact.style.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  height: 100vh;
+  align-items: center;
+  flex-flow: column wrap;
+  justify-content: center;
+`;
+
+export const H1 = styled.h1`
+  font-weight: bold;
+  font-size: 3rem;
+  background-color: var(--highlight);
+  margin-bottom: 3%;
+`;
+export const H3 = styled.h3`
+  font-weight: bold;
```

Comment:
again, this can be a styled H2.
Also, instead of using ```bold``` we should be using the font-weight of the Figma, - this is ```black``` which if you look at the font import at the top of the global styles, translates to ```900```. Also, checking the Figma, this has a font family of ```'Work Sans', sans serif;```